src/__tests__/with-theme.js

```diff
@@ -0,0 +1,71 @@
+/* eslint func-style:0 */
+import React, {Component} from 'react'
+import {render, mount} from 'enzyme'
+import * as jestGlamorReact from 'jest-glamor-react'
```

Comment:
Because you're using the `style` prop to apply styles (which I think is actually really smart), you shouldn't need this. You can just use `toMatchSnapshot` :)